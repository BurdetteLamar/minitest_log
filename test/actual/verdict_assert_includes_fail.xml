<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert_includes?' type='fail'>
    <section name='Args=[[0], 1]'>
      <verdict method='verdict_assert_includes?' outcome='failed' id='0' message='Message 0'>
        <expected class='Array' value='[0]'/>
        <actual class='Integer' value='1'/>
        <exception class='Minitest::Assertion' message='Expected [0] to include 1.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[[:a], :b]'>
      <verdict method='verdict_assert_includes?' outcome='failed' id='1' message='Message 1'>
        <expected class='Array' value='[:a]'/>
        <actual class='Symbol' value=':b'/>
        <exception class='Minitest::Assertion' message='Expected [:a] to include :b.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <expected class='Integer' value='0'/>
    <actual class='Integer' value='0'/>
  </verdict>
</log>
