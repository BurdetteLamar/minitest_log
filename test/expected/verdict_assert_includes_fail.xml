<log>
  <summary_ verdicts='2' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_assert_includes?' type='fail'>
    <section_ name='Args=[[0], 1]'>
      <verdict_ method='verdict_assert_includes?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='Array' value='[0]'/>
        <actual_ class='Integer' value='1'/>
        <exception_ class='Minitest::Assertion' message='Expected [0] to include 1.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[[:a], :b]'>
      <verdict_ method='verdict_assert_includes?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Array' value='[:a]'/>
        <actual_ class='Symbol' value=':b'/>
        <exception_ class='Minitest::Assertion' message='Expected [:a] to include :b.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
  </section_>
</log>
